CHANGELOG.yaml

```diff
@@ -12,6 +12,21 @@ meta:
 
 changes:
 
+  - title: Add endpoint to shorten URLs
+    issues:
+      - https://github.com/databiosphere/azul/issues/66
+    upgrade:
+      - terraform
+      - deploy
+      - clients
+    notes: >
+           The `/url` endpoint was added to accept URLs and return a URL redirecting to the given URL.
+           Documentation:
+           https://github.com/DataBiosphere/azul/blob/2ab5c91d13e576a0e831298681e221ab0048bbcd/lambdas/service/app.py#L701
+           Running `make terraform` will delete the previously existing azul-storage- bucket.
+           This will fail if the bucket is not empty so manual deletion is required.
+           The storage bucket should have only been used for manifests and has a 1 day expiration policy.
```

Comment:
I think I know why you mention this (to say that the bucket is likely to be empty) but I would leave it out.